css/main.css

```diff
@@ -4925,3 +4925,7 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
 	text-shadow:none;
 }
 .tg-alertcontent .tg-btn{float:right;}
+
+.hr-statistics{
+	color: white;
+}
```

Comment:
@pradeepgangwar your new commit has been added to this PR when you pushed. The review you are seeing is showing the code at the moment of review and it'll remain same.